woocommerce/woocommerce
# Link product import/export to import/export capabilities

- show Import/Export Products buttons only if the user has import/export capability
- show Woo importer/exporter under Tools only if user has import/export capability
- require both edit_products and import/export to run those actions

fixes #19431 

Testing steps:
1. Create a weaker shop manager role with capabilities `edit_products`, `Import` and `Export`.
2. Test that Import and Export works.
3. Remove `Import` capability from the new role.
4. Observe missing `Import` button under Products and Tools admin menus.
5. Navigate to `.../wp-admin/edit.php?post_type=product&page=product_importer` to check that user cannot perform the action by knowing the Import link.
6. Repeat with `Export` capability.
